iocore/net/SSLSecret.cc

```diff
@@ -55,12 +55,14 @@ SSLSecret::loadFile(const std::string &name, std::string &data_item)
   struct stat statdata;
   // Load the secret and add it to the map
   if (stat(name.c_str(), &statdata) < 0) {
+    Debug("ssl_secret", "File does not exist or cannot be found: %s", name.c_str());
```

Comment:
Perhaps a bike shed comment... I'm struggling to think of the difference between doesn't exist and "cannot be found". Would "cannot be read" (for instance a file ownership or permissions issue) describe an issue caught in this if statement?